app/controllers/api/v1/users_controller.rb

```diff
@@ -4,20 +4,20 @@ class Api::V1::UsersController < ApplicationController
   def create
     user = User.new(user_params)
     if !user.email.include?('@')
-      render json: {error: "Error creating user. A valid email must be provided."}, status: 400
+      render json: {error: "Error creating subscriber. A valid email must be provided."}, status: 400
     elsif user.save
       UserMailer.registration(user).deliver_now
-      render json: {success: "User created. A confirmation email has been sent to #{user.email}."}, status: 200
+      render json: {success: "You are now registered to receive notifications about upcoming elections in your state. A confirmation email has been sent to #{user.email}."}, status: 200
     else
-      render json: {error: "Error creating user."}, status: 400
+      render json: {error: "Error creating subscriber."}, status: 400
     end
   end
 
   def destroy
     user = User.find(params[:id])
     user.destroy
 
-    render json: {success: "Successfully unsubscribed. User will no longer recieve emails at #{user.email}."}, status: 200
+    render json: {success: "You have successfully unsubscribed from Mi Voto, Mi Voz's email list. You will no longer receive email notifications at #{user.email}."}, status: 200
   end
```

Comment:
Similar case here - adjust tests to match new success message

Also create sad path if a user can't be found